client/src/controllers/CountController.ts

```diff
@@ -0,0 +1,65 @@
+import { Controller } from '@hotwired/stimulus';
+import { ngettext } from '../utils/gettext';
+
+export class CountController extends Controller<HTMLFormElement> {
+  static values = {
+    min: { type: Number, default: 0 },
+    total: Number,
+    containerBody: { type: String, default: 'body' },
+    target: {
+      type: String,
+      default:
+        '.error-message,.help-critical,[aria-invalid="true"], .error, .w-field--error',
+    },
```

Comment:
```suggestion
    containerBody: { type: String, default: 'body' },
    min: { type: Number, default: 0 },
    target: {
      type: String,
      default:
        '.error-message,.help-critical,[aria-invalid="true"], .error, .w-field--error',
    },
    total: Number,
```

Arrange the namings in alphabetical order